mdpow/workflows/dihedrals.py

```diff
@@ -254,38 +258,24 @@ def get_bond_indices(mol, atom_indices):
     return bond_indices
 
 def get_dihedral_groups(solute, atom_indices):
-    '''Uses the indices of the relevant dihedral atom groups determined
-       by :func:`~mdpow.workflows.dihedral.get_atom_indices`
-       and returns the names for each atom in each group.
-       
-       Requires an MDPOW project directory and `resname` 
-       as input. Expands upon usage of
-       :func:`~mdpow.workflows.dihedral.get_atom_indices`
-       to return an array of the names of each atom within
-       its respective dihedral atom group as identified by
-       the SMARTS selection string. Not necessary
-       for automation, useful for verifying all atom groups of interest
-       are properly identified before analysis.
+    '''Uses the 0-based `atom_indices` of the relevant dihedral atom groups
+       determined by :func:`~mdpow.workflows.dihedrals.get_atom_indices`
+       and returns the 1-based index names for each atom in each group.
+
+       Requires the `atom_indices` from :func:`~mdpow.workflows.dihedrals.get_atom_indices`
+       to index the `solute` specified by :func:`~MDAnalysis.core.groups.select_atoms`
+       and return an array of the names of each atom within its respective
+       dihedral atom group as identified by the SMARTS selection string.
 
        :keywords:
 
-       *dirname*
-           Molecule Simulation directory. Loads simulation files present in
-           lambda directories into the new instance. With this method for
-           generating an :class:`~mdpow.analysis.ensemble.Ensemble` the
-           lambda directories are explored and
-           :meth:`~mdpow.analysis.ensemble.Ensemble._load_universe_from_dir`
-           searches for .gro, .gro.bz2, .gro.gz, and .tpr files for topology,
-           and .xtc files for trajectory. It will default to using the tpr file
-           available.
+       *solute*
```

Comment:
Just state that this is an MDAnalysis AtomGroup for the solute — be precise with the data type